samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -111,14 +111,10 @@ h4.samigo-category-subhead-2 {
   padding:.3em
 }
 
-/*
-.navModeQuestion
-{background-color:#CCC;text-align:left;padding:.3em;vertical-align:middle font-size:1.2ems}
-*/
 .navViewAction {
-background-color:#CCC;
+background-color: var(--sakai-background-color-4);
 text-align:left;
-padding:.3em;
+padding: 10px;
```

Comment:
Should this use 12 or 16 px to be inline with our other spacing ratios?

Reply:
Good point. I'll take another look at this.